src/Razor/src/Microsoft.VisualStudio.LanguageServer.ContainedLanguage/DefaultLSPDocumentSynchronizer.cs

```diff
@@ -131,10 +131,16 @@ internal void DocumentManager_Changed(object sender, LSPDocumentChangeEventArgs
                     {
                         var virtualDocument = lspDocument.VirtualDocuments[i];
 
-                        Debug.Assert(_virtualDocumentContexts.ContainsKey(virtualDocument.Uri));
+                        if (!_virtualDocumentContexts.TryGetValue(virtualDocument.Uri, out var virtualDocumentContext))
```

Comment:
It looks like these changes are already in main?

Reply:
Via https://github.com/dotnet/aspnetcore-tooling/pull/3377

Reply:
Not sure why it's showing up in diffs.